Snakefile

```diff
@@ -421,7 +432,7 @@ if augment_corpus:
         log: f"{log_dir}/split_mono_trg.log"
         conda: "envs/base.yml"
         threads: 1
-        input: f"{clean}/mono.{trg}.gz"
+        input: f"{clean}/mono.{trg}.gz", bin=deduper
```

Comment:
extract_lex doesn't depend on deduper, right? why did you add this?

Reply:
I think this is just an issue of the way the diff is displayed in the preview:
```python
if augment_corpus:
    checkpoint split_mono_trg:
        message: "Splitting monolingual trg dataset"
        log: f"{log_dir}/split_mono_trg.log"
        conda: "envs/base.yml"
        threads: 1
-        input: f"{clean}/mono.{trg}.gz"
+        input: f"{clean}/mono.{trg}.gz", bin=deduper
```
It's actually in `checkpoint split_mono_trg:`

Reply:
right